pkg/controllers/node/termination/controller.go

```diff
@@ -91,21 +91,19 @@ func (c *Controller) finalize(ctx context.Context, node *corev1.Node) (reconcile
 		return reconcile.Result{}, nil
```

Comment:
Should we add some testing for the no NodeClaim and duplicate NodeClaim cases?

Reply:
Good call, done

Reply:
Completed.